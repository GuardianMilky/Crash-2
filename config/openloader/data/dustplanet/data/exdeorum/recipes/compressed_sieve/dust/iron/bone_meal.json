{"type":"exdeorum:compressed_sieve","ingredient":{"tag":"exdeorum:compressed/dust"},"mesh":"exdeorum:iron_mesh","result":"minecraft:bone_meal","result_amount":{"type":"minecraft:binomial","n":7.0,"p":0.12}}