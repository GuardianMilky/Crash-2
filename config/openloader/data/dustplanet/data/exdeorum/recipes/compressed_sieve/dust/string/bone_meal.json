{"type":"exdeorum:compressed_sieve","ingredient":{"tag":"exdeorum:compressed/dust"},"mesh":"exdeorum:string_mesh","result":"minecraft:bone_meal","result_amount":{"type":"minecraft:binomial","n":7.0,"p":0.1}}