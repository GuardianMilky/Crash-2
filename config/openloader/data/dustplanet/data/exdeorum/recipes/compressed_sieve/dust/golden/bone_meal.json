{"type":"exdeorum:compressed_sieve","ingredient":{"tag":"exdeorum:compressed/dust"},"mesh":"exdeorum:golden_mesh","result":"minecraft:bone_meal","result_amount":{"type":"minecraft:binomial","n":7.0,"p":0.11}}